### ISSUE TITLE
Show coderdojo.com emails in Salesforce

### ISSUE THREAD

**Issue by tangentfairy** [CoderDojo/community-platform#687]
From a call with Davey it sounds like we don't put any emails from converted Dojos that are coderdojo.com emails into Salesforce.

**Comment by daveymoran-nearform** [comment]
fixed.
will update labels when fix is deployed.

**Comment by tangentfairy** [comment]
This issue seems to be resolved

**Comment by tangentfairy** [comment]
There is an issue here - This dojo https://eu2.salesforce.com/00Qb000000I3FYO?srPos=0&srKp=00Q  has a coderdojo.com email address listed in Salesforce.

In Zen, no coderdojo.com email address and none in the db https://zen.coderdojo.com/dashboard/dojo/ie/dublin/east-wall-dojo

I checked Google Admin and the coderdojo.com email was never created. So it sounds like something failed here - was it due to this update? 

Also I thought that a coderdojo.com email address would only be created when a Dojo is verified - there's no need for the Dojo to have one before we have confirmed they are legit.
